README.md

```diff
@@ -120,6 +120,21 @@ Before you begin, you should have already downloaded the Android Studio SDK and
 
 9. If the app was built successfully, you can test it by running it on either a real device or an emulated one by going to *Run > Run 'app'* or pressing the Run icon in the toolbar.
 
+    - *Note:* Disable automatic initialization for all components in case of runtime error `Unable to get provider androidx.startup.InitializationProvider` on running app on device or emulator.
+
+     - add dependency
+     `implementation "androidx.startup:startup-runtime:1.0.0"`
+
+     - include in manifest
+     ```<merge>
+     <application>
+     <provider
+            android:name="androidx.startup.InitializationProvider"
+            android:authorities="${applicationId}.androidx-startup"
+            tools:node="remove" />
+     </merge>
```

Comment:
I think the application tag close first then

Reply:
I have made the required change

Reply:
I have committed the required change